server/app/app.py

```diff
@@ -388,19 +388,13 @@ def __init__(self, app_config):
                 self.app.register_blueprint(dataroot_resources.blueprint)
 
                 self.app.add_url_rule(
-                    f"/{url_dataroot}/<dataset>",
-                    f"dataset_index_{url_dataroot}",
-                    lambda dataset, url_dataroot=url_dataroot: dataset_index(url_dataroot, dataset),
-                    methods=["GET"],
-                )
-                self.app.add_url_rule(
-                    f"/{url_dataroot}/<dataset>/",
```

Comment:
Per the [Rule Format](https://werkzeug.palletsprojects.com/en/2.0.x/routing/#rule-format), this is superfluous.

Reply:
Actually, it turns out this _has_ to be removed for testing with the multi dataset paradigm. Explained in the comments added above.